packages/@okta/vuepress-site/guides/sign-users-out/sign-out-of-okta/aspnetcore/remotesignout.md

```diff
@@ -0,0 +1,36 @@
+ASP.NET Core hosts an internal sign out handler which we called `/signout/callback`, you need to add it as a valid **Logout redirect URI** in Okta:
```

Comment:
which we called `/signout/callback` -> which is at `/signout/callback` URI?
"which we called" sounds a bit odd.
It's also better to split this in 2 sentences.
You need to add it as a valid **Logout redirect URI** in Okta: should be a new sentence.

Reply:
Yeah, I agree it sounds a bit odd. @susanharper-okta any idea to word this better?

Reply:
ASP.NET Core hosts an internal sign-out handler at `/signout/callback`. You need to add it as a valid **Logout redirect URI** in Okta: